tests/inverted-index-tests.js

```diff
@@ -220,15 +255,31 @@ describe('inverted index api', () => {
   });
 
   describe('/api/search', () => {
-    it('should return correct index when searching after creating index', (done) => {
+    it('should return correct index when searching for single file after creating index', (done) => {
       request
         .post('/api/create')
         .attach('files', `${baseDirectory}/fixures/validFile.json`)
-        .expect(200, done);
-
+        .end(() => {
+          request
+            .post('/api/search')
+            .send({ 'validFile.json': ['this is the third world second of all'] })
```

Comment:
I think this should be ['this', 'is', 'the', 'third', 'world', 'second', 'of', 'all']. Instead of putting all the words as a single element of array